### PR TITLE
Popover

### PR DESCRIPTION
**Pull request by alcferreira**
affects: @crave/farmblocks-popover

ISSUES CLOSED: #392

### CODE REVIEW

File: packages/popover/src/Popover.story.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      />
+    ))
+  )
+  .add(
+    "Custom padding",
+    withInfo()(() => (
+      <Popover
+        trigger={<button>Trigger</button>}
+        content={dismiss => (
+          <div>
```
Comment (fczuardi) [inline packages/popover/src/Popover.story.js]: I think you missed the custom padding :)

Comment (alcferreira) [reply packages/popover/src/Popover.story.js]: oops

File: packages/tooltip/src/Tooltip.js [hunk 2]
```diff
@@ -58,13 +58,16 @@ Tooltip.propTypes = {
...
+  showArrow: PropTypes.bool,
+  padding: PropTypes.string
 };
 
 Tooltip.defaultProps = {
   isVisible: true,
   align: alignments.LEFT,
-  zIndex: 1000
+  zIndex: 1000,
+  showArrow: true
```
Comment (fczuardi) [inline packages/tooltip/src/Tooltip.js]: if the default is to display the arrow, maybe the property could be a flag to hide it instead, `hideArrow` that if present hides it.

File: packages/popover/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+<a name="0.2.0-alpha.c35333c6"></a>
+# 0.2.0-alpha.c35333c6 (2018-06-15)
+
+
+### Features
+
+* **popover:** initial scaffold ([03e43dc](https://github.com/CraveFood/farmblocks/commit/03e43dc)), closes [#392](https://github.com/CraveFood/farmblocks/issues/392)
+* **Popover:** fully functional component ([eb244a6](https://github.com/CraveFood/farmblocks/commit/eb244a6))
```
Comment (fczuardi) [inline packages/popover/CHANGELOG.md]: I am assuming that this got generated because you had to manually publish the package, right?

Comment (alcferreira) [reply packages/popover/CHANGELOG.md]: yes... I shouldn't have commited but...

File: packages/popover/src/Popover.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  handleOuterClick = event => {
+    if (this.popover.contains(event.target)) {
+      return;
+    }
+
+    this.hide();
+  };
+
+  toggle = () => {
+    this.setState((prevState, props) => ({ isVisible: !prevState.isVisible }));
```
Comment (fgameiro) [inline packages/popover/src/Popover.js]: This `props` argument can be removed?

File: packages/tooltip/README.md [hunk 5]
```diff
@@ -1,25 +1,38 @@
-# Farmblocks Tooltip
+# Farmblocks popover
```
Comment (vNakamura) [inline packages/tooltip/README.md]: It seems like you got the `README.md` files from the `tooltip`  and `popover` packages mixed

Comment (alcferreira) [reply packages/tooltip/README.md]: Fixed, thanks!

**Pull request merged**